setup.cfg

```diff
@@ -67,7 +67,7 @@ filterwarnings =
 junit_family=xunit2
 
 [codespell]
-ignore-words-list=ba,blocs,coo,hist,nd,ser
+ignore-words-list=ba,blocs,coo,hist,nd,ser,mose,fo,ist
```

Comment:
where are `mose`, `fo`, and `ist` used? It might be possible to change the examples rather than ignoring them

Reply:
`mose` is a proper noun used in the `_json_normalize` example [here](https://github.com/pandas-dev/pandas/blob/master/pandas/io/json/_normalize.py#L170). It could be changed to a more traditional name to avoid this.

`fo` is used as a variable name [here](https://github.com/pandas-dev/pandas/blob/master/pandas/io/clipboard/__init__.py#L274). I do not think we need to change this.

`ist` is used as part of a valid URL [here](https://github.com/pandas-dev/pandas/blob/master/pandas/plotting/_misc.py#L157). It should not be changed.

Reply:
Thanks @zitorelova

Could you change `mose` to another name, and `fo` to `fd`?

I'll get back to you on the URL

Reply:
I've made the changes best on your comments. There was another instance of `fo` used [here](https://github.com/pandas-dev/pandas/blob/master/pandas/_testing/__init__.py#L682). I've changed some of the list entries to resolve this.